node/_tools/config.json

```diff
@@ -692,7 +693,25 @@
       "test-util.js",
       "test-vm-static-this.js",
       "test-webcrypto-sign-verify.js",
-      "test-whatwg-url-properties.js",
```

Comment:
Is this removal intentional?

Reply:
Whoops! Nope, not intentional. Fixed now. Thanks.